platforms/documentation/docs/src/docs/userguide/dep-man/how-to/how_to_share_outputs_between_projects.adoc

```diff
@@ -90,24 +101,30 @@ This configuration ensures that only the correct artifacts are exposed and preve
 [[sec:setup-consumer]]
 == Step 2: Configuring the consumer project
 
-The consumer project requests the instrumented JAR by defining matching attributes.
+In Gradle, it's important to separate the configuration where you declare dependencies from the one where you resolve artifacts.
```

Comment:
```suggestion
In Gradle, it's recommended to separate the configuration where you declare dependencies from the one where you resolve artifacts.
```

You could still use a legacy configuration to both declare and resolve (for now).